piffle/iiif.py

```diff
@@ -108,3 +108,125 @@ def format(self, image_format):
         img = self.get_copy()
         img.image_options['format'] = image_format
         return img
+
+    def parse_from_url(self, url):
```

Comment:
This seems pretty slick -  it lets you can initialize an image client instance from a url?  I think maybe it would be a bit cleaner to make it a classmethod and return a new instance.  And what do you think of calling it `init_from_url` instead of parse?